docs/resources/rds_cross_region_backup_strategy.md

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_cross_region_backup_strategy
+
+Manages RDS cross region backup strategy resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+variable "destination_region" {}
+variable "destination_project_id" {}
+
+resource "huaweicloud_rds_cross_region_backup_strategy" "test" {
+  instance_id            = var.instance_id
+  backup_type            = "all"
+  keep_days              = 5
+  destination_region     = var.destination_region
+  destination_project_id = var.destination_project_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the RDS instance.
+
+  Changing this parameter will create a new resource.
+
+* `backup_type` - (Required, String) Specifies the backup type. Value options: **auto**, **all**. Only **all** is
+  supported for SQL server.
```

Comment:
The description of the two values is missing.

Reply:
done